roles/rabbitmq/templates/etc/serverspec/rabbitmq_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+require 'socket'
+
+files= ['shadow', 'passwd']
+files.each do |file|
+  describe file("/etc/#{file}") do
+    it { should contain 'rabbitmq'}
+  end
+end  
+
+describe command('cat /etc/nova/nova.conf|grep rabbit_password') do
+  its(:stdout) { should_not contain /(rabbit_password=\w{0,7})$/ }
+end
+
+describe command('ps -ef | grep /usr/lib/rabbitmq/bin/rabbitmq-server') do
+  its(:stdout) { should contain '/usr/lib/rabbitmq/bin/rabbitmq-server'}
+end 
```

Comment:
Could you use http://serverspec.org/resource_types.html#process to check for a process?